lib/plugins/create/templates/azure-nodejs-event-hub/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "azure-nodejs",
+  "version": "1.0.0",
+  "description": "Azure Functions sample for the Serverless framework",
+  "scripts": {
+    "test": "echo \"No tests yet...\"",
+    "start": "func host start",
+    "azurite": "mkdir -p AzuriteMetadata && azurite -l AzuriteMetadata"
+  },
+  "keywords": [
+    "azure",
+    "serverless"
+  ],
+  "dependencies": {},
+  "devDependencies": {
+    "azurite": "^2.7.0",
+    "serverless-azure-functions": "^1.0.0-26"
```

Comment:
Should we just go with `"*"` as the version number here? We faced some issues in the past where users used the Google templated and got errors because the template was not updated.

Wondering if `"*"` might make sense here since you always want the most recent version when you create a template. Otherwise, if `"*"` is too wild we might want to pin it to the major version number.

@medikoo any experiences with that?

Reply:
I believe we should stick here to major version for which template was created (so `^1.0.0` in this case). If there's a major bump on a plugin, there's a risk that template will no longer work with it, so guess it's a good practice to revisit template and update it after each major bump.

I would definitely not refer to pre-release here (afaik `^` is not effective for pre-release versions.. so above range simply equals static version of`1.0.0-26`). Let's wait for official `1.0.0` release.